docs/troubleshooting.md

```diff
@@ -0,0 +1,73 @@
+# Troubleshooting
+
+If you have problems, try to rebuild when compiling with `-DCMAKE_BUILD_TYPE=Debug`.
+In particular also rebuild the index.
+The release build assumes machine written words- and docsfiles and omits sanity checks for the sake of speed.
+
+## Run End-to-End Tests
+
+QLever includes a simple mechanism for testing the entire system from
+from building an index to running queries in a single command.
+
+In fact this End-to-End Test is run on Travis CI for every commit and Pull
+Request. It is also useful for local development however since it allows you to
+quickly test if something is horribly broken.
```

Comment:
`development however since` -> `development, as`
Do we need to advertise the general benefits of good testing here?  This seems like a strange sentence in a troubleshooting documents for people trying out qlever.